sdk/storage/azure-storage-blob-nio/pom.xml

```diff
@@ -54,7 +54,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.8.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.8.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;dependency} -->
```

Comment:
This works but makes me wonder if this should be shipped from a release branch based on the `12.8.0-beta.1` release SHA.